backend/main/views/authentication.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import generics
+from ..serializers.user_serializer import UserSerializer
+from ..serializers.password_serializer import PasswordSerializer
+from ..models.user import User
+
+class RegisterView(generics.CreateAPIView):
+    queryset = User.objects.filter(is_deleted=False)
+    serializer_class = UserSerializer
+
+    def perform_create(self, serializer):
+        user = User.objects.create(
+            username=serializer.validated_data['username'],
+            email=serializer.validated_data['email']
+        )
+        user.set_password(serializer.validated_data['password'])
+        user.save()
+
+class ChangePasswordView(generics.UpdateAPIView):
+    pass
```

Comment:
Whats this view for? Will you still update this?

Reply:
Yes, I already have the updated view in the login branch that I'm currently working on.